src/controllers/user-controller.js

```diff
@@ -0,0 +1,18 @@
+import db from '../models'
```

Comment:
Not importing a "db" (database) here, but models! It's a very different thing. A model may be a representation of a table (through the ORM), or just a non-table model. Concepts are everything.